src/transform.coffee

```diff
@@ -0,0 +1,41 @@
+fs = require 'fs'
+math = require 'mathjs'
+
+die = ->
+    console.error "error:", arguments...
+    process?.exit(1)
+
+scale = (factor, m) ->
```

Comment:
Since this is dropped in as a script, it'd definitely be valuable to specify an example of the calling pattern here. Example inputs and expected output.

Reply:
Fair enough, added be25cb07244a6cdf46c97e0acc460d0d96a7f83e to clarify that this is a regex replacer function, which has Javascript-defined behavior. I just named the function `scale` and declared it here as an alternative to writing it inline, like so:

![image](https://github.com/brewingcode/shanties/assets/1566363/7792b88a-622f-42ec-b22e-d4bea3c3bc68)

...because that's way too much nesting. For example inputs and output, ef34e5d8f1887555eee807b8cc8154b54f75fd9e gives an example:

```
        cat songs/bosun-bill.txt | npm run -s scaleL 2 > foo && mv foo songs/bosun-bill.txt
```